app/models/validators/grading_system.rb

```diff
@@ -0,0 +1,8 @@
+class Validators::GradingSystem < ActiveModel::Validator
+  def validate(record)
```

Comment:
what kind of validation is this? use descriptive method name

Reply:
Hi Maam Analyn, this is my reference. Thank you https://guides.rubyonrails.org/active_record_validations.html#performing-custom-validations

![image](https://user-images.githubusercontent.com/70880887/112710114-d225d000-8ecf-11eb-9b0f-3f08f075d2fd.png)

Reply:
Oww, I see. Thank you for this ;)